scripts/start_kind_cluster.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Enable reuse of the arguments to ginkgo relevant to starting a cluster
+START_CLUSTER_ARGS=()
+for arg in "$@"; do
+  if [[ "${arg}" =~ "--kubeconfig" || "${arg}" =~ "--kubeconfig-context" ]]; then
+    START_CLUSTER_ARGS+=("${arg}")
+  fi
+done
```

Comment:
This looks wrong to me. If you invoke this with "start_kind_cluster --kubeconfig something" then it will invoke tmpnetctl with only "--kubeconfig" and not supply the "something" argument.

Might work as long as you always invoke with "--kubeconfig=something", but I've never seen commands invoked like that.

Reply:
> My shell history would like a word with you...
🤣 

This would have definitely express the intent:

```sh
if [[ "${arg}" =~ "--kubeconfig=" || "${arg}" =~ "--kubeconfig-context=" ]]; then
```